src/core/settings/searchconfig.cpp

```diff
@@ -0,0 +1,29 @@
+/*
+ * This file is part of Liri Browser
+ *
+ * Copyright (C) 2016 Tim Süberkrüb <tim.sueberkrueb@web.de>
+ *
+ * $BEGIN_LICENSE:GPL3+$
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * $END_LICENSE$
+*/
+
+#include "searchconfig.h"
+
+SearchConfig::SearchConfig(QObject *parent) : QObject(parent)
```

Comment:
Done in https://github.com/lirios/browser/pull/17/commits/f8b4bf77a577ca41086f8dc7c28524a4bcb0505c